### DISCUSSION TITLE
Port Forward Error

### DISCUSSION THREAD

**Discussion by ghost** [UseInterstellar/Interstellar#448] [General]
Hi this is James (S3CR3TASIAN) talking here, I have had a recent discussion of your interstellar proxy, it is a very nice proxy and there is one problem that has been bugging me, I use Github Codespaces but I don't know if you know how to resolve this but I tried to forward a port and change a port number to a newer port number so I can't be tracked after my link has been linked and I have tried- npm i PORT=6969 run start and I have user forwarded a port like as but it fails to automatically forward a new port containing a different port ID and it still shows as 8080, could you possibly resolve this? Perhaps I have been doing something wrong, whatever it is I really appreciate a response that can help me! Thanks! Have a good day

**Comment by 0mhx** [comment]
The correct command is `PORT=6969 npm run start`

**Comment by SchoolTARMaster** [comment]
Or go to the index.js and change the line that says `const PORT = process.env.PORT || 8080` to `const PORT = process.env.PORT || 6969`, or if that does not work change the line to `const PORT = 6969`. That is, if you want the port to be 6969 &lpar;Double Nice &colon;&rpar; &rpar;.

**Comment by ghost** [comment]
Lol, thx. I was just using it as an example 🤭. Your support is appreciated
